README.md

```
@@ -0,0 +1,5 @@
+# Deprecated
+
+This repository is deprecated.
+
+The code here has been merged into https://github.com/cloudfoundry/bosh-linux-stemcell-builder
```

markdownlint-cli2 (0.22.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)